index.html

```diff
@@ -920,5 +920,69 @@ <h3>
               </dd>
             </dl>
 </section>
+<section>
+    <h3>
+      {{RTCDataChannel}} extensions
+    </h3>
+    <p>This document extends {{RTCDataChannel}} by making it transferable.</p>
+      This allows sending and receiving messages outside the context the connection was created, for instance in workers or third-party iframes.</p>
+    <div>
+      <p>The WebIDL changes are the following:
+      <pre class="idl"
+>[Exposed=(Window,Worker), Transferable]
+partial interface RTCDataChannel {
+};</pre>
+    </div>
+    <div>
+      <p>At creation of a {{RTCDataChannel}} object, called <var>channel</var>, run the following steps:</p>
```

Comment:
We should reference https://w3c.github.io/webrtc-pc/#dfn-create-an-rtcdatachannel explicitly here.

Reply:
Done